dissect/target/plugins/os/unix/bsd/citrix/history.py

```diff
@@ -101,12 +103,15 @@ def parse_netscaler_bash_history(self, path: TargetPath) -> Iterator[CommandHist
             yield CommandHistoryRecord(
                 ts=ts,
                 command=command,
+                order=-i,  # year_rollover_helper returns entries in reverse order.
```

Comment:
Can you add a line in the docstring of `commandhistory` too, describing why it's negative?